examples/confirmation-mechanisms.js

```diff
@@ -0,0 +1,43 @@
+const { Client, CryptoTransferTransaction } = require("@hashgraph/sdk");
+
+async function main() {
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+  const operatorAccount = process.env.OPERATOR_ID;
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = new Client({
+    network: { "0.testnet.hedera.com:50211": "0.0.3" },
+    operator: {
+      account: operatorAccount,
+      privateKey: operatorPrivateKey
+    }
+  });
+
+  // Confirmation 1 - Executing will run a "precheck"
+  const transactionId = await new CryptoTransferTransaction()
+    .addSender(operatorAccount, 0)
+    .addRecipient("0.0.1", 0)
+    .setMemo("testing :)")
+    .execute(client);
+
+  // Confirmation 2 - Ask for a receipt
+  console.log("attempting to get receipt for transaction id = " + transactionId + '\n');
+  const receipt = await transactionId.getReceipt(client);
+  console.log('transaction ' + transactionId + ' receipt = ' + JSON.stringify(receipt) + '\n');
+
+  // Confirmation 3 - Ask for a record 
```

Comment:
Asking for a receipt when asking for a record is redundant. Perhaps explain that a **receipt** is what is necessary for ensure that consensus was reached. A **record** is useful only if you need the properties it provides. It has no confirmation guarantees above a **receipt**. Additionally, I would be sure to note that a **record** does cost to fetch. I don't remember the exact cost though.

Reply:
I think the code example was just to demonstrate how they would be fetched clearly - in order of detail that they provide. Hence the duplication 😄 

But I will definitely explain the context, why you would get one vs. others, and the fees. Perhaps it would just make more sense to link to those items though? e.g. fees in docs, and a blog post?